modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -3,6 +3,9 @@
 :toclevels: 1
 :php-intl-ext-url: http://php.net/manual/en/intro.intl.php
 :ppa-guide-url: https://itsfoss.com/ppa-guide/ 
+:desktop-system-requirements-url: https://doc.owncloud.com/desktop/installing.html#system-requirements
```

Comment:
@michaelstingl this is a bad thing to reference a hard html link. pls provide in the text a xref so it can be compiled correctly. Because IOS and Android are not part of docs (loaded as external source) referencing them via html is ok atm.

Reply:
Yeah, I’m aware. But desktop also lives in separate repo, and I found no quick way to link them.